rotkehlchen/tasks/events.py

```diff
@@ -226,22 +233,20 @@ def _find_asset_movement_match(
 
     if len(close_matches) == 0:
         log.debug(f'No close matches found for asset movement {asset_movement.group_identifier}')
-        return None
+        return []
```

Comment:
return close_matches
```suggestion
        return close_matches
```